src/components/Header/Logo.js

```diff
@@ -9,7 +9,7 @@ const Logo = (props) => (
     as="img"
     src={logo}
     alt="The Great British Toilet Map"
-    height={[30, 36]}
+    width={154}
```

Comment:
Avoid IE11 issue scaling logo image. I tried the min-height/width fix on the flexbox item but couldn't get anything else to work.